tests/integration/ipsec_test.py

```diff
@@ -774,24 +855,28 @@ def test_ipsec_dhcpv6_off(
           ipv4:
             enabled: true
             dhcp: true
+          ipv6:
+            enabled: true
+            dhcp: false
+            autoconf: false
           libreswan:
             hostaddrfamily: ipv6
             clientaddrfamily: ipv6
             left: {IpsecTestEnv.HOSTA_IPV6_CS}
             leftid: '@hosta.example.org'
             leftcert: hosta.example.org
-            leftmodecfgclient: no
             right: {IpsecTestEnv.HOSTB_IPV6_CS}
             rightid: '@hostb.example.org'
+            ipsec-interface: 97
```

Comment:
Why this change?

BTW, adding ipsec-interface uses a different mode in libreswan (Routing based VPN instead of Policy based VPN). Things that works well with one of them might fail with the other (probably because of bugs in NM-libreswan). Should we do all the tests in both modes maybe?

Reply:
We can only tell whether autoconf IPv6 is disabled as desired with dedicate ipsec xfrm interface.